lib/spack/docs/packaging_guide.rst

```diff
@@ -2789,13 +2798,15 @@ Adding the following to a package:
              "please use a newer release."
     )
 
-we express the fact that the current package *cannot be built* with the Intel
+expresses the fact that the current package *cannot be built* with the Intel
 compiler when we are trying to install a version "<=1.2".
 
-The ``when`` argument can be omitted, in which case the conflict will always be active.
+If the ``when`` argument is omitted then conflict will always be active.
+
+.. note::
 
-An optional custom error message can be added via the ``msg=`` parameter, and will be printed
-by Spack in case the conflict cannot be avoided and leads to a concretization error.
+    More examples of conflicts using the ``when`` clause can be found in
+    :ref:`group_when_spec`.
```

Comment:
(priority) while the linked section is called `common when= constraints`, the term "common" in this context means "when several constraints have a when cause in common" vs. "here are some when constraints that are commonly used": I don't think this section link is particularly helpful here because the only type of `when=` spec that is shown is a variant.

Reply:
Fair enough. Removing.